react-hyf/react-hyf-client/src/components/filter/GenderContainer.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { IconRandom, IconMale, IconFemale } from '../Icons';
+
+const GenderContainer = props => {
+  const isActive = gender => (gender === props.gender ? ' active' : '');
+  const [randomButton, maleButton, femaleButton] = props.buttons;
```

Comment:
Why use an array instead of an object here? Using an object it's more clear what and how you have to pass the buttons to this component I think, and the order doesn't matter in that case.

Reply:
I had some errors while I was creating ref buttons and passing them to child components. I was trying to find the error. I just forgot to change back to object destructuring. Thanks :)